app/components/works/add_files_component.html.erb

```diff
@@ -1,47 +1,49 @@
+        <fieldset>
+          <div class="dropzone dropzone-default" data-dropzone-target="container">
+            <%= form.file_field :files, multiple: true, direct_upload: true,
+                                data: { dropzone_target: 'input' } %>
+            <div class="dz-message needsclick text-secondary">
+              <p>Drop files here</p>
+            </div>
           </div>
-        </div>
-        <span style="margin: .7rem">or</span> <button type="button" class="dz-clickable btn btn-outline-primary">Choose files</button>
-        <div class="invalid-feedback" data-dropzone-target="feedback">You must attach a file<%= error_message %></div>
-        <div class="dropzone-previews" data-dropzone-target="previewsContainer">
-          <%= form.fields_for :attached_files do |file_form| %>
+          <span style="margin: .7rem">or</span> <button type="button" class="dz-clickable btn btn-outline-primary">Choose files</button>
+          <div class="invalid-feedback" data-dropzone-target="feedback">You must attach a file<%= error_message %></div>
+          <div class="dropzone-previews" data-dropzone-target="previewsContainer">
+            <%= form.fields_for :attached_files do |file_form| %>
+              <%= render Works::FileRowComponent.new(form: file_form) %>
+            <% end %>
+          </div>
+        </fieldset>
+
+        <template data-dropzone-target='template'>
+          <%= form.fields_for :attached_files, AttachedFile.new, child_index: 'TEMPLATE_RECORD' do |file_form| %>
             <%= render Works::FileRowComponent.new(form: file_form) %>
           <% end %>
-        </div>
-      </fieldset>
-
-      <template data-dropzone-target='template'>
-        <%= form.fields_for :attached_files, AttachedFile.new, child_index: 'TEMPLATE_RECORD' do |file_form| %>
-          <%= render Works::FileRowComponent.new(form: file_form) %>
-        <% end %>
-      </template>
+        </template>
+      </div>
     </div>
   </div>
   <div class="form-check pt-3" data-complex-radio-target="selection">
-    <%= form.radio_button :globus, true, class: 'form-check-input',
-      data: { action: 'complex-radio#disableUnselectedInputs' }
+    <%= form.radio_button :globus, true, class: 'form-check-input', 'data-file-uploads-target': 'globusRadioButton',
+      data: { action: 'complex-radio#disableUnselectedInputs file-uploads#updatePanelVisibility' }
     %>
-    <%= form.label :globus, 'I have uploaded my files to a Globus endpoint provided to me by SDR staff.', class: 'form-check-label' %>
-    <p>Do not check this box unless you have already discussed this with SDR staff.</p>
+    <%= form.label :globus, 'I have uploaded my files to a Globus endpoint provided to me by SDR staff.', class: 'form-check-label', style: 'font-weight: bold' %>
+    <p><em>Only select this option if you have already discussed Globus with SDR staff.</em></p>
   </div>
 </section>
```

Comment:
there are fewer changes than this diff suggests...basically added some stimulus targets and actions to the radio button, rewording text and bolding labels, and then wrapping the main file uploads panel in a new div so it can be hidden or shown as needed (which caused spacing  issues, making it appears there are number of other changes).